packages/manager/modules/exchange/src/domain/add/domain-add.html

```diff
@@ -475,6 +483,15 @@ <h3 data-translate="exchange_tab_TASKS_ADD_DOMAIN"></h3>
                     data-ng-bind="('exchange_tab_domain_add_step3_configuration_' + ctrl.model.srvParam) | translate"
                 ></dd>
 
+                <dt
+                    data-ng-if="ctrl.model.domainType !== ctrl.nonOvhDomain"
```

Comment:
You have OVH_DOMAIN, nonOvhDomain and isOvhDomain. Maybe we can reuse or standardized to avoid getting lost between affirmation and double negation.

Reply:
`isOvhDomain` doesn't exist actually that's why the code looks not unfiorm. But Ok I'll do it

Reply:
fixed !